CHANGES.rst

```diff
@@ -76,6 +76,9 @@ Bug Fixes
 - Fixed bug which did not update all references to a viewer's ID when
   updating a viewer's reference name. [#2479]
 
+- Deleting a subset while actively editing it now deselects the subset tool,
```

Comment:
The change log entry is under general category but only cubeviz viewer calls your new method. I don't quite understand.

Reply:
Works otherwise but I am still confused why only Cubeviz needs patching.

Reply:
Cubeviz is the only viewer that overrides the parent class `_on_subset_delete`, so it needs the `super()` call to trigger this new code. The others get it automatically.